apps/api/src/report/report.controller.ts

```diff
@@ -123,4 +123,34 @@ export class ReportController {
     this.logger.log(`Report: Applicants in Immigration stage (from ${from} to ${to})`);
     return this.reportService.getImmigrationApplicants(from, to);
   }
+
+  @ApiOperation({
+    summary: 'Report 8: Number of Internationally Educated Nurse Registrants Working in BC',
+  })
+  @Get('/applicant/ha-current-period-fiscal')
+  @RouteAcceptsRoles(
+    ValidRoles.HEALTH_AUTHORITY,
+    ValidRoles.HEALTH_MATCH,
+    ValidRoles.MINISTRY_OF_HEALTH,
+  )
+  async getApplicantHAForCurrentPeriodFiscal(@Query('to') to: string): Promise<object[]> {
+    this.logger.log(`Report: Number of Internationally Educated Nurse Registrants Working in BC`);
```

Comment:
Could we use interceptor or middleware for these loggings?

Reply:
@sunghwan-park-fw 
What do you suggest?
Like add a middleware that log http request(url, query param, etc)

Reply:
Yes, including response logging.

Reply:
@sunghwan-park-fw 
Let me add a separate PR for the same. will that be fine?